dbt_subprojects/daily_spellbook/models/superfluid/arbitrum/superfluid_arbitrum_flow_distribution_updated_events.sql

```diff
@@ -0,0 +1,54 @@
+{{ config(
+    schema = 'superfluid_arbitrum',
+    alias = 'flow_distribution_updated_events',
+    materialized = 'incremental',
```

Comment:
Hi @leopk-code,
for `incremental` spells, 
we need to define the `unique_key` in config blocks.

Reply:
dbt docs:
https://docs.getdbt.com/docs/build/incremental-models#defining-a-unique-key-optional